docs/source/miscellaneous_environment_variables.rst

```diff
@@ -0,0 +1,15 @@
+.. _miscellaneous_environment_variables:
+
+Miscellaneous Environment Variables
+===================================
+.. list-table::
+   :header-rows: 1
+
+   * - Variable
+     - Description
+   * - ``TORCH_FORCE_WEIGHTS_ONLY_LOAD``
+     - If set to [``1``, ``y``, ``yes``, ``true``], the torch.load will use ``weight_only=True``.
```

Comment:
Can you flesh out what this means in terms of security/why this is desirable? cc'ing @mikaylagawarecki for this one too

Reply:
Perhaps linking to `torch.load` docs would be alright since `weights_only` is defined there